addon/-private/relay.js

```diff
@@ -9,21 +9,26 @@ export default EmberObject.extend({
   changeset: null,
   key: null,
 
+  init() {
+    this._changedKeys = [];
```

Comment:
This is just a list so that they can be properly notified when a change has occurred. And how can two relays modify the same end property? This is only storing the key of the object.